pkg/clients/aws/s3/s3.go

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
Surprising we didn't have license headers on these files.  We probably missed that due to all the false positives in SonarCloud, all the more reason to fix #678 in the future :)

Reply:
Yep! Not really related to this PR, but I just noticed them and went ahead and added